### PR TITLE
Block translation

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Resources/translations/SonataBlockBundle.de.xliff [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+	<file source-language="en" target-language="de" datatype="plaintext" original="SonataBlockBundle.en.xliff">
+		<body>
```
Comment (OskarStark) [inline Resources/translations/SonataBlockBundle.de.xliff]: the indention looks wrong

**Comment by core23** [comment]
@rande @Soullivaneuh Can you help me with the French translation?

File: Resources/translations/SonataBlockBundle.de.xliff (hunk 1 above)
Comment (core23) [inline Resources/translations/SonataBlockBundle.de.xliff]: Replaced tabs with whitespaces

**Comment by OskarStark** [comment]
ping @sagikazarmark

if you don't have time, please just ignore those pings, thank you

**Comment by sagikazarmark** [comment]
``` xml
<?xml version="1.0"?>
<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
    <file source-language="en" datatype="plaintext" original="file.ext">
        <body>
            <trans-unit id="sonata.block.service.container">
                <source>sonata.block.service.container</source>
                <target>Konténer</target>
            </trans-unit>
            <trans-unit id="sonata.block.service.text">
                <source>sonata.block.service.text</source>
                <target>Szöveg</target>
            </trans-unit>
            <trans-unit id="sonata.block.service.rss">
                <source>sonata.block.service.rss</source>
                <target>RSS hírfolyam</target>
            </trans-unit>
            <trans-unit id="sonata.block.service.menu">
                <source>sonata.block.service.menu</source>
                <target>Menü</target>
            </trans-unit>
            <trans-unit id="sonata.block.service.template">
                <source>sonata.block.service.template</source>
                <target>Sablon</target>
            </trans-unit>
        </body>
    </file>
</xliff>
```

np @OskarStark

**Comment by sagikazarmark** [comment]
You might have to fix the file headers for hu lang, it is copied from the en version.

**Comment by OskarStark** [comment]
Thank you @sagikazarmark

**Comment by OskarStark** [comment]
When the French translations are added, we can merge this PR

**Comment by rande** [comment]
sonata.block.service.container => Conteneur
sonata.block.service.text => Texte
sonata.block.service.rss => Flux RSS
sonata.block.service.menu => Menu
sonata.block.service.template => Vue Partielle

**Comment by core23** [comment]
Thanks. Now we are complete :)

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**